cpp/tests/groupby/group_mean_test.cpp

```diff
@@ -14,22 +14,36 @@
  * limitations under the License.
  */
 
+#include <initializer_list>
+#include <iterator>
 #include <tests/groupby/groupby_test_util.hpp>
 
 #include <tests/utilities/base_fixture.hpp>
 #include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/type_list_utilities.hpp>
 #include <tests/utilities/type_lists.hpp>
 
 #include <cudf/detail/aggregation/aggregation.hpp>
+#include <type_traits>
+#include <vector>
+#include "cudf/utilities/traits.hpp"
 
 namespace cudf {
 namespace test {
 template <typename V>
 struct groupby_mean_test : public cudf::test::BaseFixture {
 };
 
-using supported_types = cudf::test::Types<int8_t, int16_t, int32_t, int64_t, float, double>;
+template <typename Target, typename Source>
+std::vector<Target> convert(std::initializer_list<Source> in)
```

Comment:
I thought `fixed_width_column_wrapper` already does this.

Reply:
not for double. `duration_s(duration_s::rep)` exists. `duration_s(double)` does not exist.